statement-table/src/lib.rs

```diff
@@ -46,14 +46,17 @@ pub type Summary = generic::Summary<Hash, Id>;
 pub trait Context {
 	/// Whether a authority is a member of a group.
 	/// Members are meant to submit candidates and vote on validity.
-	fn is_member_of(&self, authority: &ValidatorId, group: &Id) -> bool;
+	fn is_member_of(&self, authority: ValidatorIndex, group: &Id) -> bool;
 
-	// requisite number of votes for validity from a group.
+	/// requisite number of votes for validity from a group.
 	fn requisite_votes(&self, group: &Id) -> usize;
+
+	/// Returns id of the validator corresponding to the given index.
+	fn index_to_id(&self, index: ValidatorIndex) -> Option<ValidatorId>;
```

Comment:
Why is the `index_to_id` function needed here? `statement-table` doesn't appear to use it